polynote-kernel/src/main/scala/polynote/kernel/lang/scal/ScalaInterpreter.scala

```diff
@@ -76,6 +87,67 @@ class ScalaInterpreter(
     }
   }
 
+  private def importCellInstance(sym: global.ModuleSymbol) = {
+    val symType = global.exitingTyper(sym.toType)
+    val instanceSym = global.exitingTyper(symType.decl(global.TermName("INSTANCE")))
+    val instanceTyp = global.exitingTyper(instanceSym.info.finalResultType)
+
+    kernelContext.runInterruptibleIO(importToRuntime.importSymbol(sym)).flatMap {
+      runtimeSym =>
+        kernelContext.runInterruptibleIO {
+          try {
+            val moduleMirror = runtimeMirror.reflectModule(runtimeSym.asModule)
+            val moduleInst = runtimeMirror.reflect(moduleMirror.instance)
+            val runtimeInstanceSym = moduleInst.symbol.info.decl(runtime.universe.TermName("INSTANCE")).asTerm.accessed.asTerm
+            val instanceFieldMirror = moduleInst.reflectField(runtimeInstanceSym)
+            val instMirror = runtimeMirror.reflect(instanceFieldMirror.get)
+            (instMirror, instanceTyp, runtimeSym)
+          } catch {
+            case err: ExceptionInInitializerError =>
+              val e = err
+              throw RuntimeError(err.getCause)
+            case err: Throwable =>
+              val e = err
```

Comment:
left over from debugging?

Reply:
yeah, the `val e = error` seems to be necessary to look at the exception when debugging. But the try/catch itself is supposed to be there (unwrap `ExceptionInIinitializerError` to get the real cause).

I think I'll just leave the `val e = error` there because it doesn't really hurt anything other than looking a bit ugly. Chances are I will need breakpoints there in perpetuity 😝